x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/patch_rules.ts

```diff
@@ -108,6 +108,7 @@ export const patchRules = async ({
     id: rule.id,
     data: {
       tags: addTags(tags ?? rule.tags, rule.params.ruleId, immutable ?? rule.params.immutable),
+      throttle: null,
```

Comment:
I think this will overwrite whatever value was previously stored.  Do they always use `null` for throttle?

Reply:
Great catch! I will fix it based on siem proposal.

Reply:
I'm wondering if you can pass `undefined`, and make use of elasticsearch partial update - this gets complicated when we use schema-config, and ESO's.  Ugggh.